lib/weekly_commits/cli.rb

```diff
@@ -38,6 +43,7 @@ def weekly_commits
         committer = options[:show_author] ? ' (%cn)'.magenta : ''
 
         commits = `git --no-pager log --after='#{git_date_format} 00:00' --before='#{git_date_format} 23:59' --pretty=format:'%s#{committer}'`
+        commits += ' --no-merges' if options[:no_merge]
 
```

Comment:
Oh oops :0 I didn't notice that this will not actually work... the backticks means ruby will execute the command and return the output from it into the commits variable.

So currently this change made the output look like this if the option was used:

<img width="508" alt="image" src="https://user-images.githubusercontent.com/551858/91781401-d8fd9600-ebbf-11ea-968a-10dcfa4c7727.png">


Maybe instead you need to interpolate it into the command like I'm doing for `git_date_format` for example.

Reply:
Yep, I'm working on this right now